package/typings/index.d.ts

```diff
@@ -1,6 +1,8 @@
 import * as React from 'react';
 import * as ReactNative from 'react-native';
 
+type Constructor<T> = new (...args: any[]) => T;
```

Comment:
Hello @alpha0010, thank you for contributing!
I do see the general reasoning behind this, but I can't figure out the need to change this for this specific package that is currently running on react-native v0.69.1.
Are you able to elaborate on why changing this here? Or maybe does the original implementation currently affects your app that is running on version you mentioned (v0.71)?

Reply:
Hi! 

I think I know what's the reason behind this PR. 

After upgrading to RN 0.71 my IDE has started highlighting the same issue as @alpha0010 had mentioned.

Basically it looks like this:
<img width="657" alt="image" src="https://user-images.githubusercontent.com/6790512/214570592-af54dd9a-d483-4e97-9e11-5290961e1bf6.png">


After applying the suggested change it goes away.

Reply:
RN 0.71 bundles TS types https://reactnative.dev/blog/2023/01/03/typescript-first#declarations-shipped-with-react-native , which hides this type from general export https://github.com/facebook/react-native/blob/v0.71.1/types/private/Utilities.d.ts .

Reply:
Same problem here

Reply:
Oh yeah, so indeed:
> Or maybe does the original implementation currently affects your app that is running on version you mentioned (v0.71)?

Is the case here.
Alright!

Reply:
Same issue